pages/ssj/[workflow]/index.js

```diff
@@ -616,11 +606,6 @@ const SSJ = () => {
         open={addPartnerModalOpen}
         team={team}
       />
-      <ViewEtlsModal
-        toggle={() => setViewEtlsModalOpen(!viewEtlsModalOpen)}
-        open={viewEtlsModalOpen}
-        etls={currentETLs}
-      />
```

Comment:
Do we need this still?

https://trello.com/c/wwN9za5z/948-hide-the-other-etls-section-on-the-ssj-index?search_id=8987377f-e689-4311-b36c-1c3836a6e30f

Reply:
I'd imagine here the goal is to reinstate the modal when the data is corrected — there isn't anything wrong with this component itself and will likely be needed in the future. `main`, `staging` and `dev` all have the section that contains this component commented out re: that ticket.

Reply:
that people api request is used for this. I can comment it out and when we reinstate it, we should add pagination, so we're not requesting all the people.